### DISCUSSION TITLE
Viikon 22 päivitys

### DISCUSSION THREAD

**Discussion by AnneliO** [KohaSuomi/Koha#1250] [Tiedotteet]
Viikon 22 päivitys tehdään ma-iltana 27.5.2024 ja pitää sisällään seuraavat muutokset tai korjaukset:

## Uutta käyttäjätunnusta tehdessä virheilmoitus palauttaa kirjastovalinnan oletukseksi

Kun tekee uutta asiakastunnusta ja tallentaessa tulee jonkinlainen virheilmoitus, palautuu käyttäjän kirjastovalinta oletukseksi eli kirjautuneen käyttäjän kirjastoksi. Esim. Jos tallentaa käyttäjätunnuksen, joka on riittävän samanlainen jo olemassa olevan tunnuksen kanssa, Koha pyytää varmistamaan, että ei ole tekemässä tuplatietuetta (Kopioi asiakastiedot -kysymys). Jos on tekemässä uutta tunnusta, tulee helposti tallennettua tunnus väärään kirjastovalintaan.

![263203119-0867bb20-5cc3-43a8-b7b8-92e9443cb2dc](https://github.com/KohaSuomi/Koha/assets/33121325/ca1435a7-31b3-4fc7-ae8f-f7c7f2042909)

Tämä on nyt korjattu niin, että virheilmoitukset eivät enää palauta kirjastovalintaa oletuksesi.

Liittyy tikettiin [Koha #736](https://github.com/KohaSuomi/Koha/issues/736)

Asiasanat: uuden tunnuksen lisääminen, tuplatarkistus, kirjastovalinta

## Tarratulostuksessa Itse tulostetut -listalta pois vanhemmat niteet

Tarratulostus-liitännäiseen on lisätty ominaisuus, että yli 9 viikkoa vanhat jo tulostetut niteet poistuvat Itse tulostetut -listalta automaattisesti, kun käyttäjä avaa työkalun. Osalla käyttäjistä on ehtinyt kertyä listalle niteitä useamman vuoden ajan ja lista ei enää avautunut niteiden suuren määrän vuoksi.

![kuva](https://github.com/KohaSuomi/Koha/assets/33121325/8e1f0b45-6cef-4e28-934f-856d9d7199b0)

Liittyy tikettiin [koha-plugin-visual-label-tool #3](https://github.com/KohaSuomi/koha-plugin-visual-label-tool/issues/3)

Asiasanat: tarratulostustyökalu, itse tulostetut -lista

## EDItX-hankinnan kautta tehdyillä tilausriveillä ei näkynyt tilaajaa

Kun EDItX-hankinnan kautta tehtyjä tilauksia vastaanotti, näkyi Tilaaja-kohdassa _Tili on poistettu_. Tämä johtui siitä, että tilausriville ei tallentunut EDItX-hankinnan käyttäjätunnusta. Rajapintaa on nyt muutettu niin, että myös tilausriville tallentuu EDItX-hankinnan käyttäjätunnus.

Ennen korjausta:
![332075295-d77ca2b9-405f-44fd-b5c1-5888f0e98975](https://github.com/KohaSuomi/Koha/assets/33121325/4014a567-da10-4d0e-a469-143b2eb49c4c)

Korjauksen jälkeen:
![332361878-b28261a3-a2a7-420c-976c-85fe1fb7b7f1](https://github.com/KohaSuomi/Koha/assets/33121325/31cb814e-a941-4b3b-ae93-54956d6e0aea)

Liittyy tikettiin [Koha #1217](https://github.com/KohaSuomi/Koha/issues/1217)

Asiasanat: EDItX-hankinta, tilaaja, tilaajatieto

## Kohan itsepalvelulainauksen palautuksessa kentän nimessä väärä suomennos

Kohan itsepalvelulainauksen palautus-kentässä oli väärä suomennos lauseelle _Please enter barcode:_. Se oli kääntynyt muotoon _Kirjastokortin numero:_. Nyt se on muutettu muotoon _Lue viivakoodi:_.

Ennen muutosta:
![332407990-d78ccc1b-54f5-4828-9b35-35bcda036a09](https://github.com/KohaSuomi/Koha/assets/33121325/1961274f-3332-481d-a4be-4ddd3149b1c9)

Muutoksen jälkeen:
![kuva](https://github.com/KohaSuomi/Koha/assets/33121325/58c6d807-6f73-4a66-bbde-62ceb4c43389)

Liittyy tikettiin [Koha-translations #45](https://github.com/KohaSuomi/Koha-translations/issues/45)

Asiasanat: Kohan itsepalvelulainaus, palautus, käännösvirhe

**Comment by emta001** [comment]
Tiedotteesta poiketen, "[Uutta käyttäjätunnusta tehdessä virheilmoitus palauttaa kirjastovalinnan oletukseksi](https://github.com/KohaSuomi/Koha/issues/736)" ei ole vielä tuotu tuotantoon. Korjauksen kehityshaara unohdettiin tuoda tuotantoon, mutta se saadaan matkaan ensi viikon päivityksessä.
